integration/build/build_userns_linux_test.go

```diff
@@ -118,8 +120,11 @@ func TestBuildUserNamespaceValidateCapabilitiesAreV2(t *testing.T) {
 		container.WithImage(imageTag),
 		container.WithCmd("/sbin/getcap", "-n", "/bin/sleep"),
 	)
+
+	poll.WaitOn(t, container.IsStopped(ctx, clientNoUserRemap, cid), poll.WithDelay(100*time.Millisecond))
 	logReader, err := clientNoUserRemap.ContainerLogs(ctx, cid, containertypes.LogsOptions{
 		ShowStdout: true,
+		ShowStderr: true,
```

Comment:
We don't need this. I did it in the WIP to see if it printed something else, but I think we can drop it